client/src/components/Footer/ParticlesComponent.js

```diff
@@ -0,0 +1,94 @@
+import Particles from "react-tsparticles";
+
```

Comment:
This is a component that enables the animation of the particles moving in the background. More info [here](https://github.com/matteobruni/tsparticles/tree/main/components/react).